.github/workflows/tag-release.yml

```diff
@@ -109,12 +107,15 @@ jobs:
             --pretty=format:"- %s (%aL)" \
             "$PREVIOUS_VERSION..$CURRENT_VERSION~1" \
               | grep -v "#infra" \
-              | tee -a release.txt
+              | tee release.txt
+
+          # drop line ending
+          truncate -s -1 release.txt
```

Comment:
I'm thinking about more robust solution in case there won't be a ending newline but -- naah that shouldn't happen. :D

Reply:
There was that tempting perl one-liner somewhere on stack overflow...